packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -44,8 +46,10 @@ export default async function (rta: RuntimeAuthoring) {
         }
     );
 
+    //get all sync views
+    const syncViewsIds = getAllSyncViewsIds();
     // initialize fragment content menu entry
-    initDialogs(rta);
+    initDialogs(rta, syncViewsIds);
 
     // initialize extension point service
```

Comment:
Maybe these three comments can be removed, they do not add any value. The methods are self-explanatory enough I think.